coalib/bears/requirements/PythonRequirement.py

```diff
@@ -5,6 +5,37 @@ class PythonRequirement(PackageRequirement):
     """
     This class is a subclass of ``PackageRequirement``, and helps specifying
     requirements from ``pip``, without using the manager name.
+
```

Comment:
Isnt an easier way to simply do:
```
>>> class PyRequirement(PackageRequirement): pass
>>> # Then these doctests 
```
in the earlier place ?
If you need to do it here, it shouldn't be a doctest, as the documentation is confusing rather than helping.